### DISCUSSION TITLE
Como executar deploy dos projetos

### DISCUSSION THREAD

**Discussion by didiraja** [frontendbr/forum#684] [Discussão]
Lendo o fórum aqui, percebi que o pessoal costuma fazer o deploy direto do git para o ftp, já vi até alguns comentando que upar pelo ftp é algo antiquado.

Poderiam me explicar como se faz? Por incrível que pareça, ouvi pouquíssimo sobre isso até agora em meus estudos e o que eu vi, peguei no ar.

Existem outras opções além do git?

**Comment by caiogaspar** [comment]
Olá @didiraja, sim existem muitas opções de automatizar o seu deploy.
Automatizar o deploy é muito recomendável pois evita de fazer algumas besteiras no FTP, ex: esquecer de upar arquivos, excluir arquivos por engano, mudar local de pastas sem perceber.
Eu atualmente uso o https://deploybot.com/ para algumas aplicações, por ser muito simples de configurar e possuir integração com os repositórios mais famosos.

**Comment by rubenmarcus** [comment]
Esse artigo é sensacional !!

https://css-tricks.com/deployment/

**Comment by douglas-castilho** [comment]
se quiser tem as maneias mais simples. Com o **GULP** dá para eliminar a parte do ftp 
Um exemplo do que pode ser feito:
[gulp e ftp](https://imasters.com.br/front-end/gulp-e-ftp-atualizar-um-site-on-the-fly/ )
